### PR TITLE
Add enable logging FAQ

### PR DESCRIPTION
**Pull request by alarbada**
### Description

Adds "enable logging" FAQ.

### Quick checks:

- [x] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
- [x] There is no other [pull request](https://github.com/ConduitIO/conduit-connector-sdk/pulls) for the same
  update/change.
- [ ] I have written unit tests.
- [x] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: README.md
Comment (alarbada) [reply README.md:108]: Yup, that looks much better. Let's wait until #108 gets merged to add this then.

Comment (lovromazgon) [reply README.md:108]: Merged 👍

**Comment by alarbada** [comment]
@lovromazgon that should be it right?

**Review by lovromazgon — APPROVED** [review]

**Pull request merged**
